migration/composite_mass.py

```diff
@@ -50,7 +51,8 @@ def test_composite_mass(total_time=30):
             for key, data in wc_update.items():
                 wcecoli_timeseries[key][index] = data
         wcecoli_keys = set(wc_update.keys())
-        both_keys = (wcecoli_keys & vivarium_keys) - {'instantaniousGrowthRate'}
+        # Growth rate fluctuates a lot from timestep to timestep
+        both_keys = (wcecoli_keys & vivarium_keys) - {'instantaneous_growth_rate'}
```

Comment:
This was a funny typo that I'm sure I wasn't the first to notice but was too lazy to fix until now (it's been in wcEcoli for years).